amundsen_application/api/search/v0.py

```diff
@@ -107,35 +113,19 @@ def _create_url_with_field(*, search_term: str, page_index: int) -> str:
 @action_logging
 def _search_user(*, search_term: str, page_index: int) -> Dict[str, Any]:
     """
-        call the search service endpoint and return matching results
-        :return: a json output containing search results array as 'results'
+    call the search service endpoint and return matching results
+    :return: a json output containing search results array as 'results'
 
-        Schema Defined Here: https://github.com/lyft/
-        amundsensearchlibrary/blob/master/search_service/api/search.py
+    Schema Defined Here: https://github.com/lyft/
+    amundsensearchlibrary/blob/master/search_service/api/search.py
 
-        TODO: Define an interface for envoy_client
-        """
+    TODO: Define an interface for envoy_client
+    """
 
     def _map_user_result(result: Dict) -> Dict:
-        return {
-            'type': 'user',
-            'active': result.get('active', None),
-            'birthday': result.get('birthday', None),
-            'department': result.get('department', None),
-            'email': result.get('email', None),
-            'first_name': result.get('first_name', None),
-            'github_username': result.get('github_username', None),
-            'id': result.get('id', None),
-            'last_name': result.get('last_name', None),
-            'manager_email': result.get('manager_email', None),
-            'name': result.get('name', None),
-            'offboarded': result.get('offboarded', None),
-            'office': result.get('office', None),
-            'role': result.get('role', None),
-            'start_date': result.get('start_date', None),
-            'team_name': result.get('team_name', None),
-            'title': result.get('title', None),
-        }
+        user_result = dump_user(load_user(result))
```

Comment:
do you have any exception handling inside load_user method?

Reply:
There's a simple try-catch block inside `load_user` and `dump_user`. 

Seems like there should be a single function to decorate and cleanup a user dictionary. I'm not very familiar with this particular pattern in Python, but do you think this is the best pattern for this?